frontend/js/views/list-annotation.js

```diff
@@ -221,11 +221,11 @@ define(
              * @param  {event} event Event object
              */
             saveEnd: function (event) {
-                var $target = $(event.currentTarget),
-                    value = $target.val(),
-                    radix = 10, // Radix is 10 for decimal
-                    values,
-                    seconds;
+                var $target = $(event.currentTarget);
+                var value = $target.val();
+                var radix = 10; // Radix is 10 for decimal
+                var values;
+                var seconds;
```

Comment:
Super minor thing: From the changes I've inferred that you like to declare variables where they're initialized whenever possible. This is the only place in the code where you broke that rule. Any reason for this?

Reply:
As far as I can tell I didn't write that code; I only touched it in some linting changes. I could have gone further, sure.

So yeah, no, no particular reason.